holmes/plugins/toolsets/opensearch/opensearch_query_assist.py

```
@@ -0,0 +1,86 @@
+            type="string",
+            required=True,
+            items=ToolParameter(
+                type="object",
+                properties={
+                    "id": ToolParameter(type="string", required=True),
+                    "content": ToolParameter(type="string", required=True),
+                    "status": ToolParameter(type="string", required=True),
+                },
+            ),
+        ),
+    }
```

Ruff (0.13.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)